api/src/main/java/io/opentelemetry/trace/Sampler.java

```diff
@@ -58,4 +59,29 @@ boolean shouldSample(
    * @since 0.1.0
    */
   String getDescription();
+
+  /**
+   * Sampling decision returned by {@link Sampler#shouldSample(SpanContext, Boolean, TraceId,
+   * SpanId, String, List)}.
+   *
+   * @since 0.1.0
+   */
+  interface SamplingDecision {
+
+    /**
+     * Return sampling decision whether span should be sampled or not.
+     *
+     * @return sampling decision.
+     * @since 0.1.0
+     */
+    boolean isSampled();
+
+    /**
+     * Return tags which will be attached to the span.
+     *
+     * @return attributes which will be added to the span.
```

Comment:
Do we want to say that implementation are allowed to add them only for the spans that turn the decision from false to true?

Reply:
It makes sense, at least we could make a recommendation with `SHOULD`